docs/index.handlebars

```diff
@@ -116,7 +116,8 @@ limitations under the License.
             <div class="level-item has-text-centered">
               <div>
                 <p class="heading">Library</p>
-                <h3 class="title">{{fullName}} <span class="library-version">{{results.libraryVersion}}</span></h3>
+                <h3 class="title"><a href="{{repo.html_url}}">{{fullName}}</a> <span
+                    class="library-version">{{results.libraryVersion}}</span></h3>
```

Comment:
Nit and non actionable here. I wonder if there is some way to shorten this or link this to the github version. For example this is very long on React experimental.

Maybe even truncate it with css?w